setup-environment

```diff
@@ -20,113 +20,91 @@
 # Add options for the script
 # Copyright (C) 2019 Analog Devices, Inc.
 
-CWD=`pwd`
+CWD=$(pwd)
```

Comment:
just curious, is there an advantage over using backticks? zsh compatibility maybe?

Reply:
I'm using `shellcheck` Linux util. Please see below the result when this tool parses the origin script.

```
➜  lnxdsp-scripts git:(develop/yocto-3.1.0) shellcheck ./setup-environment 

In ./setup-environment line 23:
CWD=`pwd`
    ^---^ SC2006: Use $(...) notation instead of legacy backticked `...`.
```